targets/targets.json

```diff
@@ -4698,6 +4698,18 @@
         "features": ["BLE"],
         "release_versions": ["2", "5"]
     },
+    "ARM_MUSCA_A1": {
+        "inherits": ["Target"],
+        "core": "Cortex-M33-NS",
+        "device_has": ["INTERRUPTIN", "LPTICKER", "SERIAL", "SLEEP", "USTICKER"],
+        "macros": ["MBED_TZ_DEFAULT_ACCESS=1", "__STARTUP_CLEAR_BSS", "MBED_FAULT_HANDLER_DISABLED", "CMSIS_NVIC_VIRTUAL", "LPTICKER_DELAY_TICKS=1"],
+        "extra_labels": ["ARM_SSG", "MUSCA_A1"],
+        "supported_toolchains": ["ARMC6", "GCC_ARM"],
```

Comment:
@0xc0170 Isn't IAR required for a `release_version` of `"5"` as well?

Reply:
IAR Is required for release_version `5` for all ARMv6M and ARMv7M devices.  Only ARMC6 is `must` for ARMv8M.

Reply:
I believe it was done : https://github.com/ARMmbed/mbed-os/blob/master/tools/build_api.py#L188